packages/bot-web-ui/src/components/dashboard/tutorial-tab/quick-strategy-content/quick-strategy-cards.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { Icon, Text } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import { STRATEGIES } from 'Components/quick-strategy/config';
+import QuickStrategyGuidesDetail from './quick-strategy-guides-details';
+
+type Tcontent = {
+    qs_name: string;
+    content: string[];
+    type: string;
+};
+
+type TQuickStrategyGuides = {
+    quick_strategy_content: Tcontent[];
+    tutorial_selected_strategy: string;
+    setTutorialSelectedStrategy: (value: string) => void;
+};
```

Comment:
Maybe it would be better to rename it in some other way, we've already had the type with the name **TQuickStrategyGuides** in `packages/bot-web-ui/src/components/dashboard/tutorial-tab/config/index.ts. `

Can we somehow highlight in the name that it is the same type but with different data types? Let's think about the best way to deal with such situations...

Reply:
Renamed 👍